src/main/java/com/three_tech_solutions/slot_app/services/implementations/UserServiceImpl.java

```diff
@@ -81,4 +85,9 @@ public List<PlanResponse> getUserPlans(UUID userId) {
                 )
                 .orElseThrow(() -> new ResponseStatusException(BAD_REQUEST, "Hubo un error al encontrar el usuario"));
     }
+
+    @Override
+    public ListSlotsResponse getSlotsByDayOfWeek(UUID userId, DayOfWeek dayOfWeek) {
+        return slotService.getSlotsByDayOfWeek(getUserByIdOrThrowException(userId).getId(), dayOfWeek);
```

Comment:
Aca para que obtenemos el usuario para obtener el ID si ya nos estan pasando el ID desde el front?


```suggestion
        return slotService.getSlotsByDayOfWeek(userId, dayOfWeek);
```

Reply:
Era para corroborar que ese id exista y sino te tira la excepción, como lo validas sino?